<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dlg_Time</class>
 <widget class="QDialog" name="Dlg_Time">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>229</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>系统时间设置</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
color: #c2c2c3;
background: #343639;
}

QDialog {
border:1px solid black;
border-radius:1px;
}

#titleFrame {
border: no;
min-height: 20px;
background: url(&quot;:/Res/image/title_bar.png&quot;);
}

QPushButton {
border: no;
outline: no;
min-width: 50px;
min-height: 50px;
image: url(&quot;:/Res/image/button_50x50_normal.png&quot;);
}

QPushButton:pressed {
color: black;
border: no;
outline: no;
min-width: 50px;
min-height: 50px;
image: url(&quot;:/Res/image/button_50x50_active.png&quot;);
}

QPushButton[objectName=&quot;btnOK&quot;] {
border: no;
outline: no;
min-width: 80px;
min-height: 50px;
image: url(&quot;:/Res/image/button_80x50_normal.png&quot;);
}

QPushButton[objectName=&quot;btnOK&quot;]:pressed {
color: black;
border: no;
outline: no;
min-width: 80px;
min-height: 50px;
image: url(&quot;:/Res/image/button_80x50_active.png&quot;);
}

QPushButton[objectName=&quot;btnQuit&quot;] {
border: no;
outline: no;
min-width: 80px;
min-height: 50px;
image: url(&quot;:/Res/image/button_80x50_normal.png&quot;);
}

QPushButton[objectName=&quot;btnQuit&quot;]:pressed {
color: black;
border: no;
outline: no;
min-width: 80px;
min-height: 50px;
image: url(&quot;:/Res/image/button_80x50_active.png&quot;);
}

QDateEdit {
  color: black;
background: qlineargradient(
  x1:0, y1:0, x2:0, y2:1,
  stop:0 gray,
  stop: 0.2 white
  stop:1 white
 );
 border-radius: 1px;
 border: 1px solid black;
min-height: 46px;
}

QDateEdit::up-button {
width: no;
}

QDateEdit::down-button {
width: no;
}

QTimeEdit {
  color: black;
background: qlineargradient(
  x1:0, y1:0, x2:0, y2:1,
  stop:0 gray,
  stop: 0.2 white
  stop:1 white
 );
 border-radius: 1px;
 border: 1px solid black;
min-height: 46px;
}

QTimeEdit::up-button {
width: no;
}

QTimeEdit::down-button {
width: no;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QFrame" name="titleFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,0">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>9</number>
         </property>
         <item>
          <widget class="QPushButton" name="btnDateSub">
           <property name="text">
            <string>-</string>
           </property>
           <property name="autoRepeat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="de">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>24</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDateAdd">
           <property name="text">
            <string>+</string>
           </property>
           <property name="autoRepeat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1,0">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>9</number>
         </property>
         <item>
          <widget class="QPushButton" name="btnTimeSub">
           <property name="text">
            <string>-</string>
           </property>
           <property name="autoRepeat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTimeEdit" name="te">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>24</pointsize>
            </font>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::UpDownArrows</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnTimeAdd">
           <property name="text">
            <string>+</string>
           </property>
           <property name="autoRepeat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="rightMargin">
          <number>9</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnOK">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>确定</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnQuit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>取消</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
